### ISSUE TITLE
realpath: invalid option -- 'e'

### ISSUE THREAD

**Issue by greatbn** [digint/btrbk#92]
Hi.
I run btrbk on Ubuntu 14.04 so I have a problem when I run btrbk. I check realpath command on Ubuntu and Centos 7.2 are different. realpath on Ubuntu 14.04 doesn't have "-e" option. 
Please check again.

**Comment by digint** [comment]
Thanks for the bug report!

/sigh/

So ubuntu 14.04 still uses plain-old / different coreutils...
I will use `readlink -e` then, which seems to have broader support.

@greatbn can you please check the output of the following commands (on ubuntu / centos) for me:
```
mkdir -p /tmp/test/dir0
ln -s /tmp/test /tmp/testlink
readlink -e /tmp/testlink/dir0
echo $?
readlink -e /tmp/testlink/dir1
echo $?
```
Thanks

**Comment by digint** [comment]
As a workaround for this bug, you can create subvolumes for your targets, and `realpath` should not be executed.

e.g. if you have (in btrbk.conf): `target /mnt/backup/target_dir`, create a subvolume by running `btrfs sub create /mnt/backup/target_dir`.

**Comment by greatbn** [comment]
@digint 

Result on Centos 
```
[root@ganesha ~]# mkdir -p /tmp/test/dir0
[root@ganesha ~]# ln -s /tmp/test /tmp/testlink
[root@ganesha ~]# readlink -e /tmp/testlink/dir0
/tmp/test/dir0
[root@ganesha ~]# echo $?
0
[root@ganesha ~]# readlink -e /tmp/testlink/dir1
[root@ganesha ~]# 
[root@ganesha ~]# echo $?
1
[root@ganesha ~]# 

```

Result on Ubuntu
```
root@client-test:/home/saphi# mkdir -p /tmp/test/dir0
root@client-test:/home/saphi# ln -s /tmp/test /tmp/testlink
root@client-test:/home/saphi# readlink -e /tmp/testlink/dir0
/tmp/test/dir0
root@client-test:/home/saphi#  echo $?
0
root@client-test:/home/saphi# readlink -e /tmp/testlink/dir1
root@client-test:/home/saphi# echo $?
1
root@client-test:/home/saphi# 

```

They have  same result

**Comment by digint** [comment]
fixed in 796b6bd9bf70d95e799e5b084088001db9d487ef

using "readlink -e" now, which seems to have a more stable interface.

@greatbn please confirm if this is now working on your ubuntu / centos installs

sorry for the delay, I was pretty busy here with other projects...

**Comment by greatbn** [comment]
@digint I guess It will work. Recently I have to do something else so I will test it later.

**Comment by digint** [comment]
fixed in [btrbk-v0.23.3](/digint/btrbk/releases/tag/v0.23.3)
